src/testdir/test_python3.vim

```diff
@@ -4029,30 +4029,33 @@ func Test_python3_iter_ref()
       v = create_list()
       base_ref_count = sys.getrefcount(v)
       for el in v:
-          vim.vars['list_iter_ref_count_increase'] = sys.getrefcount(v) - base_ref_count
+        vim.vars['list_iter_ref_count_increase'] = sys.getrefcount(v) - base_ref_count
 
       create_dict = vim.Function('Create_vim_dict')
       v = create_dict()
       base_ref_count = sys.getrefcount(v)
       for el in v:
-          vim.vars['dict_iter_ref_count_increase'] = sys.getrefcount(v) - base_ref_count
+        vim.vars['dict_iter_ref_count_increase'] = sys.getrefcount(v) - base_ref_count
```

Comment:
not sure we should change the indent here?

Reply:
I've done it because previously there was a mix of 2-column-wide and 4-column-wide indentation.
The whole file is messy regarding indentation.
Vimscript segments use two spaces of indentation.
Python segments use 4 spaces of indentation, except when they don't...

I tried to be consistent within a single function, but if a vim maintainer says to revert, I will have no problems with that.